app/views/playlists/find.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="playlist_find">
+  <% @current_users_around.each do |u| %></br>
```

Comment:
Why don't you just only loop through users that have a playlist?  

Either way, getting the last playlist and checking if its null should be happening in a helper, controller or model; not in the view.